src/Chainweb/Pact/PactService.hs

```diff
@@ -856,16 +856,26 @@ execValidateBlock
     => BlockHeader
     -> PayloadData
     -> PactServiceM cas PayloadWithOutputs
-execValidateBlock currHeader plData =
-    -- TODO: are we actually validating the output hash here?
-    withBatch $ withCheckpointerRewind mb "execValidateBlock" $ \pdbenv -> do
-        !result <- playOneBlock currHeader plData pdbenv
-        return $! Save currHeader result
+execValidateBlock currHeader plData = do
+    -- KILLSWITCH: The logic here involving `txSilenceDates` is to be removed in
+    -- a future version of Chainweb. This rejects any non-genesis block which is
+    -- found to contain transactions.
+    --
+    now <- liftIO getCurrentTimeIntegral
+    case txSilenceDates (_blockChainwebVersion currHeader) of
+        Just (start, _) | now > start && not isGenesisBlock && not payloadIsEmpty ->
+            throwM $ BlockValidationFailure "Transactions are disabled."
```

Comment:
Should this error message contain some more details? For instance, that this is temporarily, it isn't a bug but a feature, and a date as of when transactions are being enabled.

Reply:
Agree